MyLibrary.Domain.Integration.Tests/BookRepositoryTest.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MyLibrary.Domain.Entities;
+
+namespace MyLibrary.Domain.Integration.Tests
+{
+    [TestClass]
+    public class BookRepositoryTest
+    {
+        private const string CONNECTIONSTRING = "lalala";
+
+        private const string DATABASE_NAME = "";
+        private IBookRepository _bookRepository;
+        [TestInitialize]
+        public void setup()
+        {
+
+        }
+        [TestCleanup]
+        public void Cleanup()
+        {
+
+        }
+
+        [TestMethod]
+        public void GetAll_ReturnsAllBooks()
+        {
+            var result = _bookRepository.GetAll();
+            Assert.AreNotEqual(null, result);
+        }
+        [TestMethod]
+        public void GetOne_ReturnsBookByTitle()
```

Comment:
Giving your test name a state will help differentiate what the intent of that method is  
usually we go with, MethodName_State_ExpectedResult()
Eg. GetOne_ValidTitile_ReturnsBookByTitle